script.js

```diff
@@ -0,0 +1,442 @@
+    {
+      while (
+        operatorStack.length > 0 &&
+        precedence(operatorStack[operatorStack.length - 1]) >= precedence(token)
+      ) 
+      {
+        output.push(operatorStack.pop());
+      }
+      operatorStack.push(token);
+    } 
+  }
+
+  while (operatorStack.length > 0)
+  {
+    output.push(operatorStack.pop());
+  }
+
+  const tokensi = output.join(" ");
+  const expression1 = tokensi.split(/\s+/);
+  console.log(expression1);
+  evaluatePostfix(expression1);
+}
+
+function evaluatePostfix(postfix) {
+  const stack = [];
+
+  for (const token of postfix) 
+  {
+    if (!isNaN(parseFloat(token))) 
+    {
+      stack.push(token); 
+    } 
+
+    else if (isOperator(token)) 
+    {
+      if (token !== 'sin' && token !== 'cos' && token !== 'tan' && token !== 'sqrt') 
+      {
+        if( stack.length == 1 || token == "-" )
+        {
+            var operand1 = stack.pop();
+            stack.push(-operand1);
+        }
+
+        else
+        {
+        const operand2 = stack.pop();
+        var operand1 = stack.pop();
+        const result = evaluateOperator(token, operand1, operand2);
+        stack.push(result);
+        }
+      }
+
+      else 
+      {
+        const operand2 = stack.pop();
+        const result = evaluateOperator(token, null, operand2);
+        stack.push(result);
+      }
+    }
+  }
+  const ans = document.getElementById('answer');
+  ans.value = "Ans: "+stack.pop();
+}
+
+function display()
+{
+  const myElement = document.getElementById('bb1');
+  const myElement1 = document.getElementById('c1');
+  const myElement2 = document.getElementById('hh');
+  myElement.style.display = 'flex';
+  myElement1.style.display = 'none';
+  myElement2.style.display = 'none';
+}
+
+function add()
+{
+  const myElement3 = document.getElementById('d1');
+  const myElement4 = document.getElementById('d2');
+  
+
+  if(myElement3.value == "pi" || myElement3.value == "e")
+  {
+    const myElement5 = document.getElementById('e2');
+    myElement5.style.display = 'flex';
+    return;
+  }
+  const myElement1 = document.getElementById('bb1');
+  myElement1.style.display = 'none';
+  const myElement2 = document.getElementById('c1');
+  myElement2.style.display = 'grid';
+  const myElement = document.getElementById('hh');
+  myElement.style.display = 'flex';
+
+  if(myElement3.value != "")
+  {
+    const myElement1 = document.getElementById('boxii');
+    const myElement = document.getElementById('textInput1');
+    myElement.value += myElement3.value+" ";
+    myElement1.style.display = 'flex';
+  }
+
+  myobj[myElement3.value] = myElement4.value;
+  myElement3.value = "";
+  myElement4.value = "";
+}
+
+function remove()
+{
+  const myElement1 = document.getElementById('bb1');
+  myElement1.style.display = 'none';
+  const myElement2 = document.getElementById('c1');
+  myElement2.style.display = 'grid';
+  const myElement = document.getElementById('hh');
+  myElement.style.display = 'flex';
+}
+
+function ok()
```

Comment:
what is this name? Please name it better as to what functionality this feature provides!